ccverify/schema.py

```diff
@@ -42,32 +43,42 @@ class ParameterValue:
     Fields
     ------
     parameter_name: str
-        Parameter name.
+        Name of the parameter being estimated.
+
+    best: float
+        Value for a parameter (often the median value of the posterior distribution).
```

Comment:
PEP 8 suggests a maximum of 72 characters for docstrings

Reply:
Fixed